lib/src/main/java/uk/gov/di/ipv/core/library/service/ConfigurationService.java

```diff
@@ -86,40 +90,30 @@ public SSMProvider getSsmProvider() {
         return ssmProvider;
     }
 
-    public boolean isRunningLocally() {
-        return Boolean.parseBoolean(System.getenv(IS_LOCAL));
-    }
-
-    public String getAuthCodesTableName() {
-        return System.getenv("AUTH_CODES_TABLE_NAME");
-    }
-
-    public String getUserIssuedCredentialTableName() {
-        return System.getenv("USER_ISSUED_CREDENTIALS_TABLE_NAME");
-    }
-
-    public String getAccessTokensTableName() {
-        return System.getenv("ACCESS_TOKENS_TABLE_NAME");
-    }
-
-    public String getIpvSessionTableName() {
-        return System.getenv("IPV_SESSIONS_TABLE_NAME");
+    public String getEnvironmentVariable(EnvironmentVariable environmentVariable) {
+        return System.getenv(environmentVariable.name());
     }
 
-    public String getIpvJourneyCriStartUri() {
-        return System.getenv("IPV_JOURNEY_CRI_START_URI");
+    public String get(ConfigurationVariable configurationVariable) {
```

Comment:
Minor thing here but could we maybe rename this method and the one below to something like
`getSsmParameter`

Reply:
Good idea, will do!